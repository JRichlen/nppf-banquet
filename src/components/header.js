import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import MenuIcon from '@material-ui/icons/Menu';
import {useHeaderStyles} from '../styles/header';

export const Header = () => {
    const classes = useHeaderStyles();

    return (
        <div className={classes.root}>
            <AppBar position='static'>
                <Toolbar>
                    <IconButton
                        aria-label='menu'
                        className={classes.menuButton}
                        color='inherit'
                        edge='start'
                    >
                        <MenuIcon/>
                    </IconButton>
                    <Typography
                        className={classes.title}
                        variant='h6'
                    >
                        {'NPPF Banquet'}
                    </Typography>
                </Toolbar>
            </AppBar>
        </div>
    );
};
