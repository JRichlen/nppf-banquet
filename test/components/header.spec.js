import React from 'react';
import renderer from 'react-test-renderer';
import {Header} from '../../src/components/header';

describe('Client', () => {
    test('should render', () => {
    // when
        const component = renderer.create(<Header/>);

        // then
        expect(component.toJSON()).toMatchSnapshot();
    });
});
