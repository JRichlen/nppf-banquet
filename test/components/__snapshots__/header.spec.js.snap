// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client should render 1`] = `
<div>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <WithStyles(ForwardRef(IconButton))>
        <MenuIcon />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        variant="h6"
      >
        NPPF Banquet
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
